bin/maxmind-geolite-mirror

```diff
@@ -1,30 +1,25 @@
 #!/usr/bin/env node
-
 const fs    = require('fs');
 const http  = require('http');
 const zlib  = require('zlib');
-
+const path = require('path');
 const config = require('../lib/config');
-
 function httpReqOpts () {
     return {
         method: 'HEAD',
         hostname: config.hostName,
         port: config.hostPort,
         headers: {
-            'User-Agent': config.userAgent,
+            'User-Agent': config.userAgent
```

Comment:
Please restore the [dangling comma](https://eslint.org/docs/rules/comma-dangle).